### DISCUSSION TITLE
Add source file entry in dependencies section

### DISCUSSION THREAD

**Discussion by ofirAbu42** [CycloneDX/specification#505] [Q&A]
Hi!
From my limited understanding - The current format of the `dependencies` section don't have an `SrcFile` entry.
In the scenario of generating an SBOM from a multi-manifests/locks project, there might be a pkg that is introduced via 2 locks in 2 different ways. It can be useful to know for each dependency entry - the source-file it relates to.

My questions:
1. Is this something that is planned / can be added to the official format?
2. If not - can you suggest a way for one to "manually" enrich a generated SBOM of a project with this data.

Thanks

**Comment by stevespringett** [comment]
Can you provide an example of what you're referring to?
